src/usr/local/www/system_certmanager.php

```diff
@@ -170,7 +170,9 @@
 			} else {
 				$res_key = openssl_pkey_get_private($keyout);
 				if ($res_key) {
-					openssl_pkey_export($res_key, $keyout, $_POST['exportpass']);
+					$args = array();
```

Comment:
You could do this in one line:
`$args = array('encrypt_key_cipher' => OPENSSL_CIPHER_AES_256_CBC);`
Actually if `$args` does not need to be passed by reference to `openssl_pkey_export` you don't need to define it at all, just use that array straight in the function call. But that can be more difficult to read, so it's not necessary.

Reply:
fixed